src/main/java/com/libtask/library2/entities/Book.java

```diff
@@ -0,0 +1,51 @@
+package com.libtask.library2.entities;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.Hibernate;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.Random;
+
+import static java.lang.Math.random;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Table (name = "books")
+public class Book {
+    static Random random = new Random();
+    @Id
+    String id = generateISBN();
+    String name;
+    String author;
+    String genre;
+    String userId;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || Hibernate.getClass(this) != Hibernate.getClass(o)) return false;
+        Book that = (Book) o;
+        return id != null && Objects.equals(id, that.id);
+    }
+
+    @Override
+    public int hashCode() {
+        return getClass().hashCode();
+    }
+
+    public static String generateISBN() {

```

Comment:
ISBN must not be generated - it`s standart so it need to be added into db manually